src/chainstate/burn/db/burndb.rs

```diff
@@ -799,11 +799,13 @@ impl BurnDB {
     }
 
     /// Get consensus hash from a particular chain tip's history
-    pub fn get_consensus_at<'a>(tx: &mut BurnDBTx<'a>, block_height: u64, tip_block_hash: &BurnchainHeaderHash) -> Result<ConsensusHash, db_error> {
+    /// Returns the empty consensus hash if the block height or block hash does not correspond to a
```

Comment:
This appears to return a `None` in that case, not the empty consensus hash.

Reply:
Yes, that's correct.  The documentation is wrong.

Reply:
Fixed in 48a05b2c0